lib/src/main/java/io/ably/lib/transport/Defaults.java

```diff
@@ -16,6 +16,7 @@
 	public static final int TIMEOUT_CONNECT         = 15000;
 	public static final int TIMEOUT_DISCONNECT      = 30000;
 	public static final int TIMEOUT_SUSPEND         = 120000;
+	public static int TIMEOUT_CHANNEL_RETRY			= 15000;
```

Comment:
It looks like you have tabs in here. Tabs only ever at the beginning of a line pls.